widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/config/call/Call.kt

```diff
@@ -0,0 +1,36 @@
+package com.glia.widgets.view.unifiedui.config.call
+
+import android.os.Parcelable
+import com.glia.widgets.view.unifiedui.config.base.Button
+import com.glia.widgets.view.unifiedui.config.base.Header
+import com.glia.widgets.view.unifiedui.config.base.Layer
+import com.glia.widgets.view.unifiedui.config.base.Text
+import com.google.gson.annotations.SerializedName
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class Call(
```

Comment:
I'm not sure, but an idea came to my mind that it would make the code clear if we named all style components like this: `CallStyle` or `CallScreenStyle`. Otherwise, it may be confusing: what kind of `Call`?

Reply:
I'm not sure about the naming of this part. I already changed it a few times. Maybe it would be better to name it `CallRemoteConfig`. What do you think?

Reply:
`CallRemoteConfig` is fine for me. At least better than just `Call`, I think.